testgrid/deploy/.terraform.lock.hcl

```diff
@@ -0,0 +1,71 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
```

Comment:
this is a new file - does it need to be here?

Reply:
I read about it. Yes? https://www.terraform.io/docs/language/dependency-lock.html

```
Terraform automatically creates or updates the dependency lock file each time you run the terraform init command. You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.

```